pkg/common/http.go

```diff
@@ -152,9 +176,9 @@ func SaneHttpClient() *http.Client {
 }
 
 // SaneHttpClientTimeOut adds a custom timeout for some scanners
-func SaneHttpClientTimeOut(timeOutSeconds int64) *http.Client {
+func SaneHttpClientTimeOut(timeoutMs int64) *http.Client {
 	httpClient := &http.Client{}
-	httpClient.Timeout = time.Second * time.Duration(timeOutSeconds)
+	httpClient.Timeout = time.Millisecond * time.Duration(timeoutMs)
```

Comment:
Since we're changing the units here already, could we change the input parameter to be a `time.Duration` so the user can choose? It also help readability:

```go
client := SaneHttpClientTimeOut(10 * time.Second) // 10s
client := SaneHttpClientTimeOut(10000)            // ???
```

This will also make code that imports this function not compile* when it gets updated, which is better than silently making all timeouts 1000x faster.

*I think.. hopefully Go doesn't silently promote an `int64` to a `time.Duration`

Reply:
That's a good suggestion but it looks like Go will silently interpret `int64` values as nanoseconds. As you pointed out, this means we should be careful when vendoring the change in.